src/Pmi/WorkQueue/WorkQueue.php

```diff
@@ -271,8 +271,9 @@ public function generateTableRows($participants, $app)
             if ($participant->withdrawalStatus == 'NO_USE') {
                 $row['withdrawal'] = self::HTML_DANGER . ' <span class="text-danger">No Use</span> - ' . self::dateFromString($participant->withdrawalTime, $app->getUserTimezone());
             } else {
-                $row['withdrawal'] = ''; 
+                $row['withdrawal'] = '';
             }
+            $row['withdrawalReason'] = $e($participant->withdrawalReason);
```

Comment:
I think you should also add `withdrawalReason` field in the static variable `$wQColumns` after `withdrawalTime`  [field] (https://github.com/all-of-us/healthpro/blob/develop/src/Pmi/WorkQueue/WorkQueue.php#L26)